### PR TITLE
Fix galaxy

### PR DESCRIPTION
**Pull request by khaledk2**
I have removed the requirements from `meta/mail.yml` and put them only in the requirements.yml file as @jburel did in the docker role.
https://github.com/khaledk2/ansible-role-ice/commit/422e51460699ef3dfd7d8f6239646cd07f2f2044
This was not enough as some attributes are set during runtime and failed during the test
https://github.com/khaledk2/ansible-role-ice/actions/runs/6725911216
I have moved the role section to the `resources/playbook.yml ` as @pwalczysko  suggested during our discussion today.
This did the trick. 
It passed the test in the GitHub actions.
https://galaxy.ansible.com/ui/standalone/roles/khaledk2/ice/
I have pushed the role to Galaxy. 
https://github.com/khaledk2/ansible-role-ice/actions/runs/6726150223

### CODE REVIEW

**Comment by khaledk2** [comment]
I have created a new file (`tasks/pre_tasks.yml`) and included it first in the `tasks/main.yml`  file.  This file contains the required roles and their variables using `include_role` as I think the roles section inside the `molecule/resources/playbook.yml ` is used for the `molecule `test.
It has passed the test.
I have pushed it to `Galaxy` without error.

**Comment by pwalczysko** [comment]
I can confirm that with this PR, I can also push to galaxy, see https://galaxy.ansible.com/ui/standalone/roles/pwalczysko/ice/

File: meta/requirements.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+- src: ome.deploy_archive
+- src: ome.basedeps
+- src: ome.java
+- src: ome.basedeps
```
Comment (pwalczysko) [inline meta/requirements.yml]: Why are basedeps twice in this list ?

**Review by pwalczysko — CHANGES_REQUESTED** [review]
A bit confused about the fact that basedeps are listed twice in both requirements.yml files. Nevertheless, the PR works and it fixes the Galaxy prob.

**Comment by pwalczysko** [comment]
This was tested successfully in a playbook run which pointed to ``khaledk2.ice`` role in Galaxy. The role successfully installed ice on the 134...testing machine @jburel

**Comment by pwalczysko** [comment]
The tests on https://github.com/ome/ansible-role-omero-server/pull/69 are passing https://github.com/ome/ansible-role-omero-server/actions/runs/6744418087 with this PR (via Galaxy khaledk2.ice)

**Review by pwalczysko — APPROVED** [review]

**Pull request merged**